crawlerr/services/rag_service.py

```
@@ -107,6 +347,11 @@ def chunk_text(self, text: str, metadata: Dict[str, Any] = None) -> List[Dict[st
                     'end_pos': end,
                     'word_count': len(chunk_text.split()),
                     'char_count': len(chunk_text),
+                    'token_count': len(chunk_text.split()) * 1.3 if self.tokenizer is None else (
+                        len(self.tokenizer.encode(chunk_text, add_special_tokens=False)) 
+                        if self.tokenizer_type == "qwen" 
+                        else len(self.tokenizer.encode(chunk_text))
+                    ),  # Accurate token count
                     **(metadata or {})
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)